enterprise/api_client/lms.py

```diff
@@ -150,13 +150,18 @@ def redirect_if_blocked(course_run_ids, user=None, ip_address=None, url=None):
         return None
 
 
-class EnrollmentApiClient(LmsApiClient):
+class EnrollmentApiClient(JwtLmsApiClient):
```

Comment:
Have we confirmed that no one else is using the enrollment client with the API key? If so, this change is fine. However, we may want to leave the LmsApiClient as a base class, and potentially modify the class to log any use of the API key so we can trace who is still using it.

Reply:
This definition of EnrollmentApiClient is only being used in edx-enterprise. We have changed it from EDX_API_KEY based client to JWT based client and there is no other usage of it.
All three clients which were inheriting `LmsApiClient` are no longer inheriting it we need to remove `LmsApiClient` once removal of EDX_API_KEY work is shipped.